processor/transformprocessor/internal/common/metrics.go

```diff
@@ -154,7 +154,7 @@ func (d dataPointStatements) handleExponetialHistogramDataPoints(ctx context.Con
 
 func (d dataPointStatements) handleSummaryDataPoints(ctx context.Context, dps pmetric.SummaryDataPointSlice, metric pmetric.Metric, metrics pmetric.MetricSlice, is pcommon.InstrumentationScope, resource pcommon.Resource) error {
 	for i := 0; i < dps.Len(); i++ {
-		tCtx := ottldatapoint.NewTransformContext(dps.At(i), metric, metrics, is, resource)
+    tCtx := ottldatapoint.NewTransformContext(dps.At(i), metric, metrics, is, resource, pmetric.NewScopeMetrics(), pmetric.NewResourceMetrics())
```

Comment:
We need to pass the actual ScopeMetrics and ResourceMetrics here, otherwise accessing the schema url wont work for summary datapoints.